src/Tooltip/tooltip.scss

```diff
@@ -47,91 +42,109 @@ $arrow-color: $brc-darkGrey !default;
     font-weight: 400;
   }
 
-  &:after, &:before {
-    border: solid transparent;
+  &::after,
+  &::before {
```

Comment:
is the `::` a rule? why is it useful? if it's only to support very old browser I vote to change this rule to accept a single `:`